src/Altinn.App.Core/Implementation/AppResourcesSI.cs

```diff
@@ -460,5 +460,18 @@ private byte[] ReadFileContentsFromLegalPath(string legalPath, string filePath)
 
             return null;
         }
+
+        /// <inheritdoc />
+        public string? GetFooter()
+        {
+            string filename = Path.Join(_settings.AppBasePath, _settings.UiFolder, _settings.FooterFileName);
+            string filedata = null;
+            if (File.Exists(filename))
+            {
+                filedata = File.ReadAllText(filename, Encoding.UTF8);
```

Comment:
We should consider making this async.
```suggestion
                filedata = await File. ReadAllTextAsync(filename, Encoding.UTF8);
```